docs/Compiler-Hardening-Guides/Compiler-Options-Hardening-Guide-for-C-and-C++.md

```diff
@@ -783,4 +782,17 @@ Note that the build ID does not act as a checksum for the executable or debug in
 
 Copyright 2023, OpenSSF contributors, licensed under [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/)
 
+## Appendix: List of Considered Compiler Options
+
+Many more security-relevant compiler options exist than are recommended in this guide. Some of these have been considered for inclusion, but for various reasons have, in-the-end been excluded from the set of recommended options. The following table lists options that have been reviewed and the rationale for their exclusion. 
+
+| Compiler Flag | Supported since  | Rationale |
+|---------------|------------------|-----------|
+| <span id="-Wl,-z,nodump">`-Wl,-z,nodump`</span>         | - | Single-purpose feature only supported by Solaris linkers[^nodump].
```

Comment:
"Single-purpose feature for Solaris compatibility" because technically, the binutils linker does implement it but it's useless on Linux.